doc/tutorials/namespaces.md

```diff
@@ -76,14 +72,8 @@ At this point the key database should have this structure:
 Another question you may ask yourself now is, what happens if we look up a key without providing a namespace. So let us retrieve the key **/b/c** with the -v flag in order to make _kdb_ more talkative.
 
 ```sh
-kdb get -v /b/c
-# STDOUT-REGEX: got \d+ keys
-#>  searching spec:/b/c, found: <nothing>, options: KDB_O_CALLBACK
-#>  searching proc:/b/c, found: <nothing>, options:
-#>  searching dir:/b/c, found: <nothing>, options:
-#>  searching user:/b/c, found: user:/b/c, options:
-#> The resulting key name is user:/b/c
-#> Value 2
+kdb get -v /tests/b/c
+# STDOUT-REGEX got \d keys⏎searching spec:/tests/b/c, found: <nothing>, options: KDB_O_CALLBACK⏎    searching proc:/tests/b/c, found: <nothing>⏎    searching dir:/tests/b/c, found: <nothing>⏎    searching user:/tests/b/c, found: user:/tests/b/c⏎The resulting keyname is user:/tests/b/c⏎The resulting value size is 8⏎Value 2
```

Comment:
Did the previous style (regex + regular stdout checks on multiple lines) not work?
The latter style is way easier to read, esp. in the rendered .md-file.
```suggestion
# STDOUT-REGEX: got \d+ keys⏎searching spec:/tests/b/c, found: <nothing>, options: KDB_O_CALLBACK⏎    searching proc:/tests/b/c, found: <nothing>⏎    searching dir:/tests/b/c, found: <nothing>⏎    searching user:/tests/b/c, found: user:/tests/b/c⏎The resulting keyname is user:/tests/b/c⏎The resulting value size is 8⏎Value 2
```

Reply:
Unfortunately, when using `STDOUT-REGEX` you are limited to using one single line. Mixing with regular STDOUT does not work (or at least I could not manage getting them playing nicely together).

I just realized that the version without the colon simply skips the test. I also just now realized that `\d` is not an ERE. The correct ERE for digits is using the character class `[[:digit:]]`.